backend/src/main/java/com/bakdata/conquery/models/query/preview/EntityPreviewExecution.java

```diff
@@ -186,7 +186,15 @@ private static Object renderValue(Object value, ResultType type, PrintSettings p
 		}
 
 		if (type instanceof ResultType.DateT) {
-			return CDate.toLocalDate((Integer) value);
+			// TODO this sidesteps issues with Jackson not localizing LocalDates when printing.
+			//  The proper approach would be to return LocalDates and configure Jackson/DateFormatter to request-scope localization.
+			// TODO is this actually a sneaky frontend-issue? Considering the other values are additionally rendered in the frontend?
```

Comment:
Wie war das in der alten Vorschau?

Reply:
Das hier ist die EntityPreview, bis jetzt wird einfach das LocalDate anhand der LocalDate.toString serialisiert.

Reply:
Das war etwas off-topic. Ich wollte nur in betracht ziehen wie wir es an anderer Stelle machen.
Ich meinte nur dass bei der alten Vorschau und (bei der neuen wahrscheinlich auch) auch Datums Werte in der Preview Tabelle ankommen.

Bei der alten Vorschau wird ja die CSV abgefragt, als rendert da das Backend genauso wie hier.
Wie macht ihr das bei der neuen Vorschau mit dem Arrow Ergebnis? Arrow hat einen extra DatenTyp für Date und hier müsste das Frontend es ja Rendern

Reply:
Achso, bei Arrow funktioniert das komplett anders, da kriegt das FE von Arrow schon Date Objekte. Bei den anderen Sachen: Genau deswegen denke ich ja, dass es eigentlich ein FE Bug ist, dir Währung wird ja auch als Decimal ausgegeben und dann angereichert mit Euro zeichen. Ich würde das - wenn überhaupt - nur mergen, damit das Geburtsdatum passt, und dann mit Kai besprechen, ob das machbar ist für ihn.